scripts/clean.py

```diff
@@ -1,13 +1,15 @@
-"""A helper script to "clean up" all of your notebooks and
-generated markdown files."""
+"""A helper script to "clean up" all of your notebooks and generated markdown & zip files."""
+
 import shutil as sh
 import os.path as op
 
 path_root = op.join(op.dirname(op.abspath(__file__)), '..')
 
 paths = [op.join(path_root, 'images', 'build'),
          op.join(path_root, '_site'),
-         op.join(path_root, '_build')]
+         op.join(path_root, '_build'),
+         op.join(path_root, 'assets', 'downloads')]
```

Comment:
not sure if I flagged this before, but I'm leaning towards putting all auto-generated files (including the .zip file) in `_build/downloads`. WDYT?